visual-dotnet/SauceLabs.Visual/VisualClient.cs

```diff
@@ -256,19 +163,27 @@ private async Task<string> VisualCheckAsync(string name, VisualCheckOptions opti
         }
 
         /// <summary>
-        /// <c>Cleanup</c> set a correct status to the build. No action should be made after that calling <c>Cleanup</c>.
+        /// <c>Cleanup</c> set a correct status to the build. No action should be made after calling <c>Cleanup</c>.
         /// </summary>
         public async Task Cleanup()
         {
-            if (!_externalBuild)
+            if (Build.Close != null)
```

Comment:
should we raise an exception when the close method is undefined? Maybe it would be better to do it and fail hard rather than silently fail

Reply:
Removed